### PR TITLE
Added watchServer functionality

### PR DESCRIPTION
**Pull request by MKKuhn**

### CODE REVIEW

File: handlers/watch.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  var _self = this;
+
+  this.Logger.debug("Sending watch_server CMsgWatchGame request");
+  var server64ID = new SteamID(server_steam_id.replace("=", "")).getSteamID64();
+
+  var payload = {
+    "server_steamid": server64ID,
+    "watch_server_steamid": 0,
+    "lobby_id": 0,
+    "regions": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
```
Comment (fjlaubscher) [inline handlers/watch.js]: Maybe try to retrieve this from existing config object rather than hardcoding regions.

File: handlers/watch.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  this.sendToGC(  Dota2.schema.lookupEnum("EDOTAGCMsg").values.k_EMsgGCWatchGame,
+                  Dota2.schema.lookupType("CMsgWatchGame").encode(payload).finish(),
+                  onWatchServerResponse, callback)
```
Comment (fjlaubscher) [inline handlers/watch.js]: I think there could be a missing semicolon here.

**Review by fjlaubscher — CHANGES_REQUESTED** [review]
I left a few comments. Really minor things.

**Comment by MKKuhn** [comment]
@fjlaubscher Changes made 👍 Could you check again please?

**Pull request merged**